superset/assets/src/dashboard/actions/dashboardLayout.js

```diff
@@ -213,10 +213,8 @@ export function handleComponentDrop(dropResult) {
     } else if (
       destination &&
       source &&
-      !(
-        // ensure it has moved
-        (destination.id === source.id && destination.index === source.index)
-      )
+      !// ensure it has moved
```

Comment:
It looks odd to me to have the comment in between the `!` and the `(`, seems like the kind of stuff that different JS interpreters might behave differently around...

Reply:
I'm not sure how this change got in. I think it might have been the result of autoformatting. Shouldn't be a problem for the interpreter, but I'll change it back anyway.